src/main/java/frc/robot/Constants.java

```diff
@@ -16,6 +16,8 @@
  */
 public final class Constants {
 
+  public static final int minvoltage = 10;
```

Comment:
Feel like this should start higher.  at 10.5 mininum, or 11.

from what I am seeing on ChiefDelphi, 10.5V is seen as the point where battery performance drops quickly to zero.
And we don't want the batteries to be harmed by draining them. 

 ( And because we can't shut down the robot, we should assume that there will be some current drop while a team member walks onto the field to switch the battery. )

https://www.chiefdelphi.com/t/about-battery-life/126825
https://www.chiefdelphi.com/t/cba-5-and-battery-testing/408305

Reply:
Completed, changed it to 12 for good measure